templates/gitlab.rb.j2

```diff
@@ -70,5 +70,17 @@ nginx['ssl_verify_client'] = "{{ gitlab_nginx_ssl_verify_client }}"
 nginx['ssl_client_certificate'] = "{{ gitlab_nginx_ssl_client_certificate }}"
 {% endif %}
 
+# Extra configuration
+{% if gitlab_extra_settings is defined %}
+{% for extra in gitlab_extra_settings %}
+{% for setting in extra %}
+{% for kv in extra[setting] %}
+{{ setting }}['{{ kv.key }}'] = '{{ kv.value }}'
```

Comment:
You may want to add an option to not quote numbers, or `true` / `false`.

Reply:
I agree. But I have to confess that I have no idea how to do that. Any clue?

I can suggest to remove the quotes of the `{{ kv.value }}`, and let the user to add or not them.

Reply:
```
{% if kv.value is string %}
... = '{{ kv.value }}'
{% else %}
... = {{ kv.value }}
{% endif %}
```
Messy, but works.

Reply:
I took into account string, iterable and others. Changed by 29d07dc.